plugin/src/main/java/com/exadel/aem/toolkit/plugin/utils/DialogConstants.java

```diff
@@ -149,13 +149,15 @@
     public static final String PN_TRACKING_ELEMENT = "trackingElement";
     public static final String PN_TYPE = "type";
     public static final String PN_TYPE_HINT = "typeHint";
+    public static final String PN_UPDATE_COMPONENT_LIST = "updatecomponentlist";
     public static final String PN_USE_FIXED_INLINE_TOOLBAR = "useFixedInlineToolbar";
     public static final String PN_VALUE_MEMBER = "valueMember";
     public static final String PN_VALUE_TRANSFORM = "valueTransform";
     public static final String PN_WRAPPER_CLASS = "wrapperClass";
 
     public static final String VALUE_ALL = "all";
     public static final String VALUE_NONE = "none";
+    public static final String VALUE_POLICY_RESOLVER_FORMAT = "Granite.policyResolver.build('%s')";
```

Comment:
Can't seem the candidate fo this constants class. Okay, several other constants, e.g. node names, are also used only once. But they like *potentially can* be used elsewhere. But this one is too specific for reusing

Reply:
yeah, looks like this constant should be in AllowedChildrenHandler